dist/platforms/ubuntu/steps/activate.sh

```diff
@@ -74,6 +74,26 @@ elif [[ -n "$UNITY_SERIAL" && -n "$UNITY_EMAIL" && -n "$UNITY_PASSWORD" ]]; then
   # Store the exit code from the verify command
   UNITY_EXIT_CODE=$?
 
+elif [[ -n "$UNITY_LICENSING_SERVER" ]]; then
+  #
+  # Custom Unity License Server
+  #
+  echo "Adding licensing server config"
+  pushd /resources
+  cat services-config.json.template | tr -d '\r' | awk "{sub(/%URL%/,\"$UNITY_LICENSING_SERVER\")}1" > services-config.json
+  mkdir -p /usr/share/unity3d/config/
+  mv services-config.json /usr/share/unity3d/config/
+
+  /opt/unity/Editor/Data/Resources/Licensing/Client/Unity.Licensing.Client --acquire-floating > license.txt #is this accessible in a env variable?
+  PARSEDFILE=$(grep -oP '\".*?\"' < license.txt | tr -d '"')
+  export FLOATING_LICENSE
+  FLOATING_LICENSE=$(sed -n 2p <<< "$PARSEDFILE")
+  FLOATING_LICENSE_TIMEOUT=$(sed -n 4p <<< "$PARSEDFILE")
```

Comment:
The parsing and writing of this file should happen in the logic (javascript) part, not in bash. We're trying to keep the bash part as thin as possible because it's extremely hard to test and maintain.

Reply:
I assume you are talking about the services-config.json? I can definitely fix that, but is there any way for me to copy it to the right location inside the container without mounting and doing it from bash? Or should I just mount it to /usr/share/unity3d/config/ in the docker command since I am creating the directory from script anyways?